compiler/frontend/src/builtin_functions.rs

```diff
@@ -7,53 +7,413 @@ use lazy_static::lazy_static;
-    GetArgumentCount,    // function -> argumentCount
-    IfElse,              // condition thenFunction elseFunction -> resultOfExecutedFunction
-    IntAdd,              // (summandA: int) (summandB: int) -> (sum: int)
-    IntBitLength,        // (value: int) -> (numberOfBits: int)
-    IntBitwiseAnd,       // (valueA: int) (valueB: int) -> (result: int)
-    IntBitwiseOr,        // (valueA: int) (valueB: int) -> (result: int)
-    IntBitwiseXor,       // (valueA: int) (valueB: int) -> (result: int)
-    IntCompareTo,        // (valueA: int) (valueB: int) -> (ordering: Less | Equal | Greater)
-    IntDivideTruncating, // (dividend: int) (divisor: int) -> (quotient: int)
-    IntModulo,           // (dividend: int) (divisor: int) -> (modulus: int)
-    IntMultiply,         // (factorA: int) (factorB: int) -> (product: int)
-    IntParse,            // text -> (parsedInt: maybeOfInt)
-    IntRemainder,        // (dividend: int) (divisor: int) -> (remainder: int)
-    IntShiftLeft,        // (value: int) (amount: int) -> (shifted: int)
-    IntShiftRight,       // (value: int) (amount: int) -> (shifted: int)
-    IntSubtract,         // (minuend: int) (subtrahend: int) -> (difference: int)
-    ListFilled,          // (length: int) item -> list
-    ListGet,             // list (index: int) -> item
-    ListInsert,          // list (index: int) item -> list
-    ListLength,          // list -> int
-    ListRemoveAt,        // list (index: int) -> (list, item)
-    ListReplace,         // list (index: int) newItem -> list
-    Print,               // message -> Nothing
-    StructGet,           // struct key -> value
-    StructGetKeys,       // struct -> listOfKeys
-    StructHasKey,        // struct key -> booleanTag
-    TagGetValue,         // tag -> any
-    TagHasValue,         // tag -> booleanTag
-    TagWithoutValue,     // tag -> tag
-    TextCharacters,      // text -> (listOfText: list)
-    TextConcatenate,     // (textA: text) (textB: text) -> (concatenated: text)
-    TextContains,        // text (pattern: text) -> booleanTag
-    TextEndsWith,        // text (suffix: text) -> booleanTag
-    TextFromUtf8,        // (bytes: listOfInteger) -> resultOfText
-    TextGetRange,        // text (startInclusive: int) (endExclusive: int) -> (substring: text)
-    TextIsEmpty,         // text -> (isEmpty: booleanTag)
-    TextLength,          // text -> (length: int)
-    TextStartsWith,      // text (prefix: text) -> booleanTag
-    TextTrimEnd,         // text -> text
-    TextTrimStart,       // text -> text
-    ToDebugText,         // any -> text
-    TypeOf,              // any -> typeTag
+    /// Takes two values. Returns `True` if both values are equal, otherwise
+    /// `False`.
+    ///
+    /// TODO: Document how equality works. Also, change how equality for
+    /// functions works.
+    ///
+    /// ```candy
+    /// ✨.equals 5 5 => True
+    /// ✨.equals 5 2 => False
+    /// ```
+    Equals,
```

Comment:
I think it makes more sense to move all of these comments over to the Builtins package (`packages/Builtins/_.candy`, where some builtins already have documentation)